PythonRegression/tests/features/steps/api_test_steps.py

```diff
@@ -13,52 +14,98 @@
 neighbors = static_vals.TEST_NEIGHBORS
 testAddress = static_vals.TEST_ADDRESS
 
+localVar = "TESTINGLOCALVARIABLESAAAAAAAAAAAAAAAAAAAAAAAAAAH"
+
```

Comment:
Is it a local variable if it is global?
Is it a constant? maybe you want to write it as upper case as a reminder that it shouldn't change?

Reply:
This shouldn't have been left in the file and has been since removed